src/plugins/intel_gpu/src/graph/include/loop_inst.h

```diff
@@ -88,12 +88,12 @@ struct typed_program_node<loop> : public typed_program_node_base<loop> {
         auto params = parent::get_kernel_impl_params(in_layouts, out_layouts);
         params->inner_progs = { get_primitive()->body_program };
         // Set memory_deps using custom get_memory_deps to add current_iteration(mutable_data) into memory_deps
-        params->memory_deps = get_memory_deps();
+        params->memory_deps = get_const_memory_deps();
```

Comment:
Why only setting const mem deps to memory_deps?

Reply:
Fixed it. thanks.